### PR TITLE
Add dark melodic rp script.

### PR DESCRIPTION
**Pull request by miraco**
This isnt finished yet. 
Current Bugs:

- Have to figuer out how often he does this movement and change the waittime for "Dark Clearic Malod Spawn" then.
- some waypoints need adjustments
- some timers need adjustments

Im not having any sniffs anymore and no programm. I will try it with comparing videos. 
https://www.youtube.com/watch?v=zlHN6PWUHkA&feature=youtu.be
Thats a video of it how it currently works.

### CODE REVIEW

**Comment by killerwife** [comment]
Okay. A few notes. On retail currently the event is broken. So it is not too annoying, put it as smth like "once half an hour". I do not have the sniff, so I can't help there. Next, please change your SQL codestyle from:
INSERT INTO `creature_movement_template` VALUES 
INSERT INTO `creature_movement_template` VALUES 
INSERT INTO `creature_movement_template` VALUES 
to:
INSERT INTO `creature_movement_template` VALUES 
(),
(),
(),
because it then executes the queries faster. Once you do that, we will merge this PR.

File: Updates/1090_dark_melodic_script.sql [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '15', '-237.851', '1023.36', '54.309', '0', '0', '0', 'Dark Clearic Malod Waypoint');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '16', '-234.792', '1026.51', '54.3085', '0', '34000', '1943503', 'Dark Clearic Malod third Event');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '17', '-201.884', '1016.94', '54.3022', '0', '0', '0', 'Dark Clearic Malod Waypoint');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '18', '-166.062', '1017.46', '54.2065', '0', '0', '0', 'Dark Clearic Malod Waypoint');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '19', '-151.175', '1012.26', '54.183', '0', '0', '0', 'Dark Clearic Malod Waypoint');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '20', '-145.295', '1000.05', '54.2113', '0', '0', '0', 'Dark Clearic Malod Waypoint');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '21', '-140.269', '982.351', '54.2018', '0', '0', '0', 'Dark Clearic Malod Waypoint');
```
Comment (AnonXS) [inline Updates/1090_dark_melodic_script.sql]: INSERT INTO `creature_movement_template` (`entry`, `pathId`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `waittime`, `script_id`, `comment`) VALUES
. 
.
. as kw mentioned add the table structure too just as safeguard incase there is table struct differences between classic-tbc-wotlk-db. Same for `dbscript_string` & `dbscripts_on_creature_movement`

**Comment by miraco** [comment]
Will edit this when i am at home.

What you mean with broken on retail?

**Comment by killerwife** [comment]
That on BFA, me and my friend stood there for 1 hour and didnt see the event happen anymore.

**Comment by miraco** [comment]
Should be good now.
Updated timers to how it is currently working on retail. I watched it for 2 hours yesterday and it happends one in 1 hour.

The only thing currently can happen:
npc supply master broog entry 19436 is also doing an RP event on mangos core every 300000 mili seconds (5 minutes?)
I never saw him doing this on retail in the 2 hours i was watching. (But maybe its broken on retail)

So both npcs are walking to Lieutenant General Orion. It can happen that both stand there and try to talk with him.
Maybe we have to check that one of them is already doing his rp event and delay the other one.

**Pull request closed without merging**